django/db/backends/base/base.py

```diff
@@ -36,6 +37,8 @@ class BaseDatabaseWrapper(object):
     ops = None
     vendor = 'unknown'
     SchemaEditorClass = None
+    # Classes instantiated inside __init__().
+    validation_class = BaseDatabaseValidation
```

Comment:
Do you feel strongly about not including the rest of the Base classes as defaults here? That would save having to import them in the dummy backend and perhaps make it more clear all the attributes that should be overridden.